TypeProvider/Core/Prelude.fs

```diff
@@ -0,0 +1,22 @@
+[<AutoOpen>]
+module Froto.TypeProvider.Core.Prelude
+
+open System
+open System.IO
+
+open Printf
+
+/// Raises NotSupportedException with given string as message
+let notsupportedf fmt = ksprintf (NotSupportedException >> raise) fmt
```

Comment:
Sounds reasonable, I'll add one (or a few) exceptions to the TypeProvider project and later we can make them inherited from some general Froto exception type once it's introduced